### ISSUE TITLE
Request: Optional (Government?) Holidays for BC

### ISSUE THREAD

**Issue by rmclellan** [pcraig3/hols#117]
Similar to Alberta, would it be possible to add optional holidays for BC?  It appears that public sector employees take these off and it would be useful to have.

- Easter Sunday
- Easter Monday
- National Day for Truth and Reconciliation
- Boxing Day

Source: https://www2.gov.bc.ca/gov/content/health/practitioner-professional-resources/msp/claim-submission-payment/designated-holidays-and-close-off-dates

Note:  It appears that National Day for Truth and Reconciliation has yet to be set for 2022 but the government is advising it be observed.  Please see this note from the public sector union.  https://www.cupe.bc.ca/bulletin_national_day_for_truth_and_reconciliation#:~:text=Last%20year%20marked%20the%20first,as%20a%20federal%20statutory%20holiday

**Comment by pcraig3** [comment]
Hey dude, this is great, thanks so much for the PR and for adding the source!

[I ran a very scientific survey](https://github.com/pcraig3/hols/commit/5aa2c68d168a525b5f8ed23c1b362038473252cd) on the Alberta holidays page for about a week after I launched them and it came back with most people saying they want to see them, so yeah, I'm in favour. I would totally roll out optional holidays for other provinces if I could find sources for them, the problem is that literally any non-stat holiday (Halloween, Valentines Day, religious holidays) can be an optional holiday so without a source, I don't want to just start adding them based on a gut feeling. But the source you've linked to looks pretty solid, so that's perfect 👍

I'm currently on holiday but I can get to this at the weekend — looks good from my first glance so I don't anticipate any issues. 

One unanticipated programming problem I've run into recently is a little out there: due to moving out of the timezone I was in in Canada, a bunch of my stupid tests are failing, so I want to figure that out and then make sure the existing tests work on my machine and then we're good. So yeah, let's say ETA of about a week to get this in.
